### DISCUSSION TITLE
UFS_forcing_ensemble_generator.py netcdf import error in HERA

### DISCUSSION THREAD

**Discussion by AnningCheng-NOAA** [NCAR/ccpp-scm#391] [Cases, Initial Conditions, Forcing] [answered]
I get the following error when run UFS_forcing_ensemble_generator.py in HERA. Any clue to fix?

(env_ufsreplay) /scratch1/NCEPDEV/global/Anning.Cheng/ccpp-scm/scm/etc/scripts% ./UFS_forcing_ensemble_generator.py -d /scratch1/BMC/gmtb/CCPP-SCM/UFS_RTs/control_c192/ -sc --C_RES 192 -dt 360 -n control_c192 -lons 300 -lats 34
Traceback (most recent call last):
  File "./UFS_forcing_ensemble_generator.py", line 8, in <module>
    from netCDF4 import Dataset
  File "/home/Anning.Cheng/.conda/envs/env_ufsreplay/lib/python3.8/site-packages/netCDF4/__init__.py", line 3, in <module>
    from ._netCDF4 import *
ImportError: libjpeg.so.8: cannot open shared object file: No such file or directory

**Comment by dustinswales** [comment]
@AnningCheng-NOAA I was able to run your script w/o issue:
![Screen Shot 2023-06-23 at 12 30 41 PM](https://github.com/NCAR/ccpp-scm/assets/13949033/1c9e80de-7542-4e33-a779-7b75fd3e2768)
Is your python environment correct? (If you do a "conda list", you should see netcdf4)

**Comment by AnningCheng-NOAA** [comment]
Dustin:

Attached is what I have from running conda list.  netcdf is there, but I
still get the error message.

**Reply by dustinswales** [reply]
@AnningCheng-NOAA I don't see an attachment?
Can you try "conda deactivate" followed by "conda activate env_ufsreplay"?

**Reply by AnningCheng-NOAA** [reply]
I tried conda deactivate, get the same error.  Could you point me to your Hera_setup_intel.csh file? 
![conda_deac](https://github.com/NCAR/ccpp-scm/assets/48297505/93e14840-996e-4b82-9eb8-e82b00235b4b)
![conda_list](https://github.com/NCAR/ccpp-scm/assets/48297505/46cc68c5-0d81-4e7c-b585-533fa4430cf2)

**Reply by dustinswales** [reply]
It looks like you are missing a few packages... shapely, xarray, and xesmf.
See https://github.com/NCAR/ccpp-scm/discussions/390 to create the correct environment

**Comment by AnningCheng-NOAA** [comment]
Dustin:

I got a xesmf not found error.   
![xesmf_error](https://github.com/NCAR/ccpp-scm/assets/48297505/c715e529-212e-4836-8d40-a6d0fcbc6c30)

**Reply by dustinswales** [reply, marked as answer]
Try "xesmf", not "xesm"

**Reply by AnningCheng-NOAA** [reply]
Dustin:
The ufs replay is working now! Thank you and have a great weekend,
